process/smartContract/scQueryService.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-var maxGasValue = big.NewInt(math.MaxInt64)
+var maxGasValue = math.MaxInt64
```

Comment:
Please add a TODO to find a non-maxInt64 value that assure us that we wont end up executing infinite loops (caused by a bad written contract) but also assures that the vm will run the tx call. Maybe replace it with the maximum gas limit / block? If the latter value is ok, we have to inject it :D

Reply:
added. injected it.